src/Command/PluginChecksumCheckCommand.php

```
@@ -0,0 +1,132 @@
+
+        $criteria = new Criteria();
+        $criteria->addFilter(new EqualsFilter('name', $pluginName));
+        $plugin = $this->pluginRepository->search($criteria, $context)->first();
+        if ($plugin instanceof PluginEntity) {
+            $plugins->add($plugin);
+        } else {
+            $io->error(sprintf('Plugin "%s" not found', $pluginName));
+        }
+
+        return $plugins;
+        return $plugins;
```

PHPStan (2.0.3)

Unreachable statement - code above always terminates.


(deadCode.unreachable)